distributed/deploy/spec.py

```diff
@@ -11,6 +11,53 @@
 from ..security import Security
 
 
+class SpecProcess:
```

Comment:
`ProcessInterface` ?

Reply:
I guess that more clearly describes what it is. Still not sure though.

Reply:
I've made this change for now.  I've also removed this from the top level API.